src/badges.less

```diff
@@ -0,0 +1,35 @@
+.badge {
+  .border;
+
+  display: inline-block;
+  padding: .25em .4em;
+  font-size: 75%;
+  font-weight: 700;
+  line-height: 1;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  background-color: #868e96;
```

Comment:
Shouldnt this be assigned a colour variable?

Reply:
Yeah totally forgot about that, I'll replace it asap!